### DISCUSSION TITLE
docs: Complete nested object are created and just id are linked

### DISCUSSION THREAD

**Discussion by RedOperator2565** [mikro-orm/mikro-orm#5351] [General]
Hello,

I've had a little problem when i tried to create a new entity with relations. I gave MikroORM the entity with complete object as relation and it tried to create the objects each time. I didn't undertood at this time that it needs to be only the id to make a relation and not a creation. I didn't see any mention of this detail and maybe it could save time for other people in the same situation if it was added. 

Another action, would be an option to check if relation exists, create it if not or fail if not existent. 

With my thanks for this project.

**Comment by B4nan** [comment]
This is partially described here: https://mikro-orm.io/docs/entity-helper#updating-deep-entity-graph

The behavior for `em.create` is the same as for `em.assign`. I'll mention it somewhere in the getting started guide too.

> Another action, would be an option to check if relation exists, create it if not or fail if not existent.

You can't load anything from inside a sync method.

**Reply by RedOperator2565** [reply]
Thanks for the quick reply. i checked the link you gave and it did mention it.
I think it could be a good idea to add this on the quick start too. 

However, i find the 'assign' method's documentation not really clear on the reference page.

Thanks again and tell me if i can of any help.
